examples/sample-app/application-template-stibuild.json

```diff
@@ -161,7 +161,39 @@
           "replicas": 2
         },
         "strategy": {
-          "type": "Rolling"
+          "type": "Rolling",
+          "rollingParams": {
+            "pre": {
+              "failurePolicy": "Abort",
+              "execNewPod": {
+                "containerName": "ruby-helloworld",
+                "command": [
+                  "/bin/true"
+                ],
+                "env": [
+                  {
+                    "name": "CUSTOM_VAR1",
+                    "value": "custom_value1"
+                  }
+                ]
+              }
+            },
+            "post": {
+              "failurePolicy": "Ignore",
+              "execNewPod": {
+                "containerName": "ruby-helloworld",
+                "command": [
+                  "/bin/false"
```

Comment:
Even if failures are being ignored, why do we want /bin/false as the command here?